packages/flame/test/camera/camera_component_test.dart

```diff
@@ -60,6 +60,19 @@ void main() {
       }
     });
 
+    testWithFlameGame('camera should be able to retarget follow', (game) async {
+      final world = World()..addToParent(game);
+      final camera = CameraComponent(world: world)..addToParent(game);
```

Comment:
No need to create a new world and camera here, you can use `game.camera` and `game.world`.